src/core/pagination/pagination.model.js

```diff
@@ -0,0 +1,9 @@
+import Resource from '../resource/resource';
+
+export default class PaginationModel {
+	constructor() {
+		this.resource = new Resource();
```

Comment:
current will be a part of scroll. maybe view, maybe of pagination, so for now let it be fake current)